binary_translations.pm

```diff
@@ -253,50 +253,60 @@ sub data {
+        "海上護衛艦隊(MemId:{0})は{1}のダメージ({2})" => {ok => 'a-csharp 4604361' }, #"Escort Fleet {0} deals {1} damage {2}"           ???
+        "海上護衛艦隊名海上護衛艦隊名" => { },                #"Maritime Escort Fleet Name Escort Fleet Name"       ???
+        "[000000][0055aa]海上護衛艦隊[-]の配備について" => { }, #"Maritime Escort Fleet about deployment"           ???
+
+        "廃棄は □ボタンで行います" => {ok => 'a-csharp 4517038'}, #"Press □ to discard"       delete equipment with square
+        "解体は □ボタンで行います" => {ok => 'a-csharp 4517066'}, #"Press □ to demolish"       delete ships with square
+        "ボタンで戦略コマンドを" => {},                      #"Press button for Strategy Command"          ???  figured it out, this string is missing 2 kanji ボタンで戦略コマンドを\n開き、
+        "輸送船団や海上護衛艦隊への" => {},                   #"To the Transports and Maritime Escort Fleet"      ??? tutorial to show you where to assign transports/escort fleet?
+        "建造日数が掛かります" => {},                        # "Construction days required"              ??? tutorial about building ships? alternate "Days construction will take"
         "海域の輸送船団に海上護衛艦隊を配備することで、\r\n敵通商破壊部隊から輸送船を護ることが可能です。\n\nまた" => {},
-        "備の輸送船数:{0}(この海域への移動中:{1}) - 総数:{2}" => {},
-        "輸送船 x {0}" => {},
-        "ロック済艦娘を" => {},
-        "艦隊司令部情報" => {},
-        "艦隊切替" => {},
-        "艦隊数" => {},
-        "ボタンで戦略コマンド" => {},
-        "[000000][0055aa]輸送船[-]の配備について" => {},
-        "工廠[-]での輸送船建造や、" => {},
-        "[66ccff]全艦種出撃可能[-]です。"=>{},
+       # "It is possible to protect ships from enemy raids by deploying fleets to escort transportation ships in an area."
+       # Depriciated: "By deploying an escort fleet to a transportation fleet in an area, \r\n it is possible to protect ships from an enemy raiding unit. \ N \ n"
+        "備の輸送船数:{0}(この海域への移動中:{1}) - 総数:{2}" => {}, #"Number of Transports:{0} (moving to this area: {1})  - Total number: {2} "
+        "輸送船 x {0}" => {},                             #"Transports x {0}" 
+        "ロック済艦娘を" => {},                            #"Ship Girl is locked" literal "Locked ship daughter" 
+        "艦隊司令部情報" => {},                            #"Fleet Headquarters intelligence"
+        "艦隊切替" => {},                                 #"Switch fleet"
+        "艦隊数" => {},                                   #"Fleet number"
+        "ボタンで戦略コマンド" => {},                       #"Press button for Strategy Command"
+        "[000000][0055aa]輸送船[-]の配備について" => {},    #"Transports Deployed" - My context may be off, but OC this is how I am reading it.
+        #                                                  Depriciated "About deployment of Transports" alternate "About Transport deployment"
+        "工廠[-]での輸送船建造や、" => {},                  #"Factory[-]Building a Transport ship at," alternate "Building a Transport ship at the Factory" color codes missing from string would go around Factory
+        "[66ccff]全艦種出撃可能[-]です。"=>{},              #"All classes of ships may sortie."

```

Comment:
I'm not backing down on this Type vs Class thing.

https://en.wikipedia.org/wiki/List_of_types_of_naval_vessels

Reply:
Sort by Class Link: http://kancolle.wikia.com/wiki/Ship_Class
http://en.kancollewiki.net/wiki/Ship_Class

http://kancolle.wikia.com/wiki/Ship
![2018-12-19_202922](https://user-images.githubusercontent.com/45841539/50263867-c5e76a00-03cc-11e9-8519-dc9a7b8375c3.png)

http://en.kancollewiki.net/wiki/Glossary#Class_classification
![2018-12-19_203142](https://user-images.githubusercontent.com/45841539/50263928-12cb4080-03cd-11e9-930b-ae3389702abb.png)

I should point out, that this is where you are making your mistake. Like all uneducated misconceptions, there is som small amount of truth to what you are saying. But, the truth stops as soon as light is shown.
It is true, a Destroyer is a type of ship. Just as it is true, an aircraft carrier, or a light cruiser is a type of ship.
But, a DD and a DE are different classes of ship inside the destroyer type.
A CVL, CV, CVB are some of the classes of ships inside of the aircraft carrier type.
And, let us not forget, CL, CLT, CT, are some of the classes inside the light cruiser type.

So, as the game breaks them down to the Classes within the different Types, as shown here:
![2018-12-19_204452](https://user-images.githubusercontent.com/45841539/50264327-ed3f3680-03ce-11e9-9562-892acf9583e8.png)
We are using Class.

And, stop citing Wikipedia as your only sources. Not only is it considered not acceptable as an authoritative source by Universities of good reputation, it does not even consider itself to be a source of verifiable information.

Pick another mound to spill your blood on. This one is over.

Reply:
The Ship Class link shows the various classes of ship: http://kancolle.wikia.com/wiki/Ship_Class

Destroyer
- Kamikaze-class
- Mutsuki-class
- Fubuki-class
- Ayanami-class
- Akatsuki-class
- Hatsuharu-class
- Shiratsuyu-class
- Asashio-class
- Kagerou-class
- Yuugumo-class
- Akizuki-class
- Shimakaze (unique ship with no sister ships thus not a class)

Destoyer is the Type, the Class is named after the lead ship of that class.

> A ship class is a group of ships of a similar design. This is distinct from a ship type, which might reflect a similarity of tonnage or intended use. For example, USS Carl Vinson is a nuclear aircraft carrier (ship type) of the Nimitz class (ship class). ... Ships of the same class may be referred to as sister ships.